### PR TITLE
Release 1.15.2 [rpm]

### PR DESCRIPTION
**Pull request by dLobatog**
Only for 1.15.2  - remind me when the tags are pushed so the pipeline can continue please (http://projects.theforeman.org/projects/foreman/wiki/Release_Process#Packaging-a-release)

### CODE REVIEW

File: foreman-installer/foreman-installer.spec [hunk 1]
```diff
@@ -1,12 +1,12 @@
 # set and uncomment all three to set alpha tag
-#global alphatag RC2
-#global dotalphatag .%{alphatag}
-#global dashalphatag -%{alphatag}
+#%global alphatag
+#%global dotalphatag .%{alphatag}
+#%global dashalphatag -%{alphatag}
```
Comment (ehelms) [inline foreman-installer/foreman-installer.spec]: Ah ha! This is the problem -- you can't comment out a definition directly. RPM spec will still process this. Thats why before it had dropped the `%` and I believe is the cause of the issue.

**Comment by dLobatog** [comment]
[test]

**Comment by mmoll** [comment]
merged (not built/tagged, I guess you will do this with Jenkins), gracias @dLobatog!

**Comment by dLobatog** [comment]
@mmoll It needs to be tagged, Jenkins builds from tags http://ci.theforeman.org/view/Release%20pipeline/job/release_packages/85/

**Comment by mmoll** [comment]
I gues only @ehelms can help for now then... I do have a tito VM, but not with me at the moment and didn't boot since ages...

**Pull request merged**